src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -21,7 +21,7 @@
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. Auto-capitalises first letter."
```

Comment:
Typo:
": Adds a person to the address book. Auto-capitalises first letter."
->
": Adds a person to the address book. Auto-capitalises first letter. "

Reply:
Should have a `/n` before the next sentence. Not a big deal, but proper spacing would be nice.